File: src/test/java/medium/MaximumManhattanDistanceTest.java

package medium;
import java.util.stream.Stream;
import static org.junit.jupiter.api.Assertions.*;
import org.junit.jupiter.params.ParameterizedTest;
import org.junit.jupiter.params.provider.Arguments;
import org.junit.jupiter.params.provider.MethodSource;

public class MaximumManhattanDistanceTest {
MaximumManhattanDistance instance = new MaximumManhattanDistance();
@ParameterizedTest
@MethodSource("dataMaximumManhattanDistance")
public void testMaximumManhattanDistance(int[] nums, boolean expected) {

boolean result = instance.solve(nums);

assertEquals(expected, result);
}
public static Stream<Arguments> dataMaximumManhattanDistance(){
	return Stream.of();
}
}

File: medium/maximum_manhattan_distance_test.go

package medium

import (
	"testing"
)

func TestMaximumManhattanDistance(t *testing.T) {
cases := []struct {
num 	int
expected bool
}{
{true, true},
}
for _, c := range cases {
t.Run("", func(t *testing.T) {
result := MaximumManhattanDistance(c.num)
if result != c.expected {
t.Errorf("MaximumManhattanDistance(%d) == %v, expected %v", c.input, result, c.expected)
}

})

}
}